.github/workflows/test-go.yml

```diff
@@ -166,6 +166,21 @@ jobs:
               ${{ inputs.extra-flags }} \
                 \
               ${test_packages[${{ matrix.runner-index }}]}
+      - name: Prepare datadog-ci
+        if: github.repository == 'hashicorp/vault'
```

Comment:
Is the reason that this is only run on **hashicorp/vault** because the self-hosted runners that we use on the enterprise repository already have the DataDog CLI installed?

Reply:
Yes. In hindsight, I should leave a comment with that information here.

Reply:
```suggestion
        # Enterprise CI runners already have datadog-ci installed, so we can skip this step on ENT
        if: github.repository == 'hashicorp/vault'
```

Reply:
I don't think that's necessary.